services/ui_backend_service/api/autocomplete.py

```diff
@@ -0,0 +1,110 @@
+from services.utils import handle_exceptions, web_response
+import asyncio
+import threading
+
+
+class AutoCompleteApi(object):
+    # Cache tags so we don't have to request DB everytime
+    tags = []
+
+    def __init__(self, app, db):
+        self.db = db
+        # Cached resources
+        app.router.add_route("GET", "/autocomplete/tags", self.get_all_tags)
```

Comment:
Should we also include users here (cached) so that we have all the Home requirements covered?

Reply:
Currently i'm using tags for users field as well. Do you think we should get separate these endpoints? Good thing abotu using tags is that we send only 2 initial requests on frontpage, flows and tags where tags includes project, user and tags field